index.html

```diff
@@ -62,6 +62,11 @@ <h3>Wordpress Password Generator</h3>
 				document.getElementById('password').value = randomizedPassword;
 				document.getElementById('password-hash').value = HashPassword(randomizedPassword);
 			}
+			
+			// Update hash on text change
+			document.getElementById('password').addEventListener('input', function (evt) {
+				document.getElementById('password-hash').value = HashPassword(this.value);;
```

Comment:
Can you remove this extra semicolon at the end of the line?

Reply:
Fixed in 6e77a8f9ded40eff44c4c3242267466096737ea7, good catch :+1: